scene/resources/surface_tool.cpp

```diff
@@ -958,8 +958,7 @@ void SurfaceTool::create_from_blend_shape(const Ref<Mesh> &p_existing, int p_sur
 			break;
 		}
 	}
-	ERR_FAIL_COND(shape_idx == -1);
-	ERR_FAIL_COND(shape_idx >= arr.size());
+	ERR_FAIL_INDEX(shape_idx, arr.size());
```

Comment:
It is equivalent but semantically not exactly the same, checking `== -1` explicitly was to make sure that something was found (since `shape_idx` is initialized to `-1` as the "not found" default value).
But it's probably fine either way here.

Reply:
Fixed. I avoided changing `== -1` on search snippets but I must have missed one.